### ISSUE TITLE
Overseerr loading images incredibly slowly (30s-1min)

### ISSUE THREAD

**Issue by maxwell-01** [sct/overseerr#3316]
### Description

Firstly I would like to say that I love the app and I wish I had found it sooner!

The images in overseerr are loading incredibly slowly. I have read the posts both here and on reddit and have tried multiple ways of setting it up. I have tried setting up the DNS as suggested and I have tried various network settings via docker. Currently I have this running via a VPN container with the suggested DNS added there. The application itself is snappy but every time it needs to load an image, everything hangs. This can be for 30s - 1 min on the home page as there are so many images.

### Version

1.32.5

### Steps to Reproduce

1. Install Overseerr via docker apps on Unraid
1. Open Overseerr
2. Witness slow loading

### Screenshots

![image](https://user-images.githubusercontent.com/45392468/216757425-feaab89b-7f09-48d8-901f-b54ce78ee19c.png)
![image](https://user-images.githubusercontent.com/45392468/216757475-45949096-7849-40b7-8788-a2e958bb1f2a.png)

### Logs

```shell
Nothing relevant I can see in the logs, just a lot of the following:

2023-02-04T08:05:00.327Z [info][Plex Scan]: Beginning to process recently added for library: Films {"lastScan":1675497600025}
2023-02-04T08:05:00.332Z [info][Plex Scan]: Beginning to process recently added for library: TV programmes {"lastScan":1675497600031}
2023-02-04T08:05:00.973Z [info][Plex Scan]: Recently Added Scan Complete 
2023-02-04T08:06:00.004Z [debug][Jobs]: Starting scheduled job: Download Sync 
2023-02-04T08:06:00.012Z [debug][Download Tracker]: Found 4 item(s) in progress on Sonarr server: Sonarr 
2023-02-04T08:10:01.097Z [info][Plex Scan]: Recently Added Scan Complete
```

### Platform

desktop

### Device

Windows, Mac, Android and IOS

### Operating System

Windwos 11, Mac OS

### Browser

Chrome, Safari

### Code of Conduct

- [x] I agree to follow Overseerr's Code of Conduct

**Comment by hirenshah** [comment]
How is your Docker network set up? I've been having the same issue with Overseerr on Synology Docker for absolutely ages and adding DNS servers or tweaking nginx reverse proxy settings didn't help.  After reading some helpful threads on Reddit, I changed my container network from a custom macvlan to the default bridge and now everything is so so much faster.

**Comment by maxwell-01** [comment]
Thanks for the message. I have tried bridge, host and also routing through a VPN container. I have also tried completing deleting the container and all files and setting up from scratch again with no improvement.

**Comment by mduijm** [comment]
Hey,
I had the issue as well, running Docker on Linux and fixed it by adding a DNS entry in my docker-compose file. Here is the example:

```
  overseerr:
    image: lscr.io/linuxserver/overseerr
    container_name: overseerr
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Europe/XX
    volumes:
      - /home/user/docker/overseer:/config
    ports:
      - 5055:5055
    dns:
      - <internal DNS server IP>
      - 1.1.1.1
      - 8.8.8.8
    restart: unless-stopped
```

Other users on Reddit also found this helpfull, hope this helps for you as well ;)

**Comment by maxwell-01** [comment]
Hello,

Thank you for your help. However I had already tried this with many different configurations (running via compose and the built in unraid docker config) with no use.

I've installed a m.2 ssd and set overseerr to run off that and it's improved massively. It's odd as all my other containers run from my raid array without issue.

I'm happy to report that moving this to a dedicated ssd solved the problem.
